honeybee_schema/energy/ventcool.py

```diff
@@ -93,13 +95,154 @@ class VentilationOpening(NoExtraBaseModel):
         'friction from window geometry, insect screens, etc. Typical values include '
         '0.17, for unobstructed windows with insect screens and 0.25 for unobstructed '
         'windows without insect screens. This value should be lowered if windows are '
-        'of an awning or casement type and not allowed to fully open.'
+        'of an awning or casement type and not allowed to fully open. Default:0.17.'
     )
 
     wind_cross_vent: bool = Field(
         False,
         description='Boolean to indicate if there is an opening of roughly equal area '
         'on the opposite side of the Room such that wind-driven cross ventilation will '
         'be induced. If False, the assumption is that the operable area is primarily on '
-        'one side of the Room and there is no wind-driven ventilation.'
+        'one side of the Room and there is no wind-driven ventilation. Default: False.'
     )
+
+    air_mass_flow_coefficient_closed: float = Field(
+        default=None,
+        gt=0,
+        description='A number in kg/s-m used to calculate the mass flow rate '
+        '(air_mass_flow_coefficient * dP^air_mass_flow_exponent) when the opening is '
+        'closed, defined at 1 Pa per meter of crack length. This property is only '
+        'required if running an AirflowNetwork simulation.'
```

Comment:
@chriswmackey, it'll be the former: no assumption will be made and you won't be able to simulate the AFN. This is because EP has this as  a required value, and doesn't provide a default value for it. I'll add text to clarify.

Reply:
Update: This is another one where maybe we should reference DesignBuilder's crack template. See comment below for more detail on this.